distribution/lib/Standard/Google_Api/0.0.0-dev/src/Main.enso

```diff
@@ -58,3 +67,36 @@ initialize secret_file =
     http_transport = GoogleNetHttpTransport.newTrustedTransport
     json_factory = GsonFactory.getDefaultInstance
     Google_Api_Client.Value credential json_factory http_transport
+
+
+## PLACEHOLDER, performs google analytics call
+
+   Arguments:
+   - property_id
+   - start_date
+   - end_date
```

Comment:
When we add such arguments list, it should contain short descriptions for each argument.

If the arguments are trivial and there is no need to document it - we shouldn't include the list at all as just the argument names are clear from the function signature itself, so a list without descriptions is redundant.